sdk/deviceupdate/azure-iot-deviceupdate/samples/simple_sample.py

```diff
@@ -0,0 +1,56 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# The MIT License (MIT)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the ""Software""), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+#
+# --------------------------------------------------------------------------
+import io
+import logging
+import os
+
+from azure.iot.deviceupdate import DeviceUpdateClient
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+# Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+# AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, DEVICEUPDATE_ENDPOINT, DEVICEUPDATE_INSTANCE_ID
+try:
+    endpoint = os.environ["DEVICEUPDATE_ENDPOINT"]
+    instance = os.environ["DEVICEUPDATE_INSTANCE_ID"]
+except KeyError:
+    LOG.error("Missing environment variable 'DEVICEUPDATE_ENDPOINT' or 'DEVICEUPDATE_INSTANCE_ID' - please set if before running the example")
+    exit()
+
+# Build a client through AAD
+client = DeviceUpdateClient(credential=DefaultAzureCredential(), endpoint=endpoint, instance_id=instance)
+
+try:
+    # Raise an exception if the service rejected the call
+    response = client.management.list_device_classes()
```

Comment:
The sample does not cover much breadth of the API, nor do I see a suite of tests for these changes.
Could we expand the coverage a little? Either for this preview, or open an issue to address for the next?

Reply:
To make the sample more effective, service team needs to create more resource. Maybe we could expand the coverage later.